blockchain/proxy.go

```diff
@@ -165,10 +167,51 @@ func (ep *elrondProxy) SendTransaction(tx *data.Transaction) (string, error) {
 	return response.Data.TxHash, nil
 }
 
+// SendTransactions broadcasts the provided transactions to the network and returns the txhashes if successful
+func (ep *elrondProxy) SendTransactions(txs []*data.Transaction) ([]string, error) {
+	jsonTx, err := json.Marshal(txs)
+	if err != nil {
+		return nil, err
+	}
+	buff, err := ep.PostHTTP(sendMultipleTransactionsEndpoint, jsonTx)
+	if err != nil {
+		return nil, err
+	}
+
+	response := &data.SendTransactionsResponse{}
+	err = json.Unmarshal(buff, response)
+	if err != nil {
+		return nil, err
+	}
+	if response.Error != "" {
+		return nil, errors.New(response.Error)
+	}
+
+	return ep.postProcessSendMultipleTxsResult(response)
+}
+
+func (ep *elrondProxy) postProcessSendMultipleTxsResult(response *data.SendTransactionsResponse) ([]string, error) {
+	txHashes := make([]string, 0, len(response.Data.TxHashes))
+	indexes := make([]int, 0, len(response.Data.TxHashes))
+	for index := range response.Data.TxHashes {
+		indexes = append(indexes, index)
+	}
+
+	sort.Slice(indexes, func(i, j int) bool {
```

Comment:
last time I checked, the order was already correct. do we need the sorting?

Reply:
Yes, the response comes as a map. Will be thoroughly tested in next PRs